python/cudf_polars/cudf_polars/dsl/expressions/unary.py

```diff
@@ -147,6 +149,14 @@ def __init__(
         self.name = name
         self.options = options
         self.children = children
+        self.is_pointwise = self.name not in (
+            "cum_min",
+            "cum_max",
+            "cum_prod",
+            "cum_sum",
+            "set_sorted",
```

Comment:
```suggestion
            "drop_nulls",
```
`set_sorted` asserts sortedness by fiat, rather than actually sorting the column.

But we do need drop_nulls.